### PR TITLE
adt data sets are added for obs prep

### PR DESCRIPTION
**Pull request by apchoiCMD**
lists of rad sets to the test

### CODE REVIEW

**Comment by guillaumevernieres** [comment]
@apchoiCMD :
1. You did not have to close the previous PR
2. Choose a title that describe the work that you want to commit to develop.  **fixed conflict file merging from team members** does not describe your work.

File: utils/test/prepdata.sh [hunk 1]
```diff
@@ -3,7 +3,16 @@ set -e
 
 project_source_dir=$1
 
+ncgen -o rads_adt_3a_2021181.nc4 ${project_source_dir}/testdata/rads_adt_3a_2021181.cdl
+ncgen -o rads_adt_3b_2021181.nc4 ${project_source_dir}/testdata/rads_adt_3b_2021181.cdl
+ncgen -o rads_adt_3a_2021182.nc4 ${project_source_dir}/testdata/rads_adt_3a_2021182.cdl
```
Comment (guillaumevernieres) [inline utils/test/prepdata.sh]: You're going to hate me @apchoiCMD ! 
Let's just commit 2 .cdl files. Sorry, I should have thought about this earlier.

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
See comment. I think 2 ```.cdl``` files are probably enough ... Sorry again!

File: utils/test/prepdata.sh (hunk 1 above)
Comment (apchoiCMD) [reply utils/test/prepdata.sh]: @guillaumevernieres only 2 files of cdl? like "rads_adt_3a_2021181.nc4" and "rads_adt_3b_2021181.nc4"?

Comment (guillaumevernieres) [reply utils/test/prepdata.sh]: Yes, sounds good @apchoiCMD

Comment (apchoiCMD) [reply utils/test/prepdata.sh]: @guillaumevernieres  Updated prepdata.sh and removed testdata files as well

**Review by guillaumevernieres — APPROVED** [review]
Perfect! Thanks @apchoiCMD .

**Pull request merged**